.rubocop.yml

```diff
@@ -60,6 +60,7 @@ RSpec/MultipleExpectations:
   Max: 5
   Exclude:
     - 'spec/features/*'
+    - 'spec/services/discovery_report_spec.rb'
```

Comment:
```suggestion
    - 'spec/**/*_spec.rb'
```

Reply:
Done, though this effectively just disables that cop I believe by ignoring any specs.